pack/sm_encounter.json

```diff
@@ -1355,12 +1375,14 @@
 		"position": 125,
 		"quantity": 1,
 		"scheme_hazard": 1,
+		"scheme_text": "Hinder 2[per_player]. \n <b> When Revealed: </b> Move the glider token to the main scheme with the most threat. Resolve thats schemes <b>Special</b> ability.",
 		"set_code": "venom_goblin",
 		"set_position": 17,
 		"type_code": "side_scheme"
 	},
 	{
 		"boost": 0,
+		"boost_text": "Resolve te","scheme_text": "Resolve the <b>Special</b> ability of the scheme with the glider counter.",
```

Comment:
I assume the boost text is supposed to be the same as the scheme text.

Reply:
whoops good catch, I will fix